demos/smart_classroom_demo_gapi/main.cpp

```diff
@@ -0,0 +1,561 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <chrono>  // NOLINT
+
+#include <gflags/gflags.h>
+#include <monitors/presenter.h>
+#include <ie_iextension.h>
+
+#include <opencv2/gapi/infer/ie.hpp>
+#include <opencv2/gapi/render.hpp>
+
+#include "smart_classroom_demo.hpp"
+#include "custom_kernels.hpp"
+#include "actions.hpp"
+#include "stream_source.hpp"
+#include "kernel_packages.hpp"
+
+using namespace InferenceEngine;
```

Comment:
Do we actually need this? I though in the G-API version IE is not used directly.